Druid/DRInterpreterToCompiler.class.st

```diff
@@ -41,6 +41,26 @@ DRInterpreterToCompiler class >> fromInterpreterClass: anInterpreterClass [
 		  yourself
 ]
 
+{ #category : #'instance creation' }
+DRInterpreterToCompiler class >> generateDruidJITModel [
+
+	<script>
+	| compilerClassName interpreterCompiler compilationUnit primitives |
+	compilerClassName := #DruidJIT.
+	primitives := #( #primitiveAdd #primitiveSubtract #primitiveLessThan
+	                 #primitiveGreaterThan #primitiveLessOrEqual
+	                 #primitiveGreaterOrEqual #primitiveEqual #primitiveNotEqual
+	                 #primitiveMultiply #primitiveDivide ).
+
+	interpreterCompiler := self
+		                       fromInterpreter:
+		                       (DRDummyInterpreter new primitives: primitives)
```

Comment:
Why DRDummyInterpreter, couldn't we use the primitive selection now?
Maybe next iteration

Reply:
Yes, we can use it now. With what filter?